plugins/item_tasks/server/rest.py

```diff
@@ -34,17 +34,41 @@ def __init__(self):
     @autoDescribeRoute(
         Description('List all available tasks that can be executed.')
         .pagingParams(defaultSort='name')
+        .param('minFileInputs', 'Search tasks by minimum number of file inputs.', required=False,
+               dataType='int', default=0)
```

Comment:
While `default=0` and `default=None` (the implicit default) have the same truthiness and effect on the code, perhaps it'd be more consistent to make them both default to `None`? I don't feel really strongly though.